swagger.yaml

```diff
@@ -0,0 +1,286 @@
+swagger: '2.0'
+info:
+  title: Analytics Core API
+  version: '0.1'
+basePath: /api/v1/
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /liveness:
+    get:
+      tags:
+        - Service settings
+      operationId: f8a_admin.api_v1.get_liveness
+      summary: Get job service liveness
+      responses:
+        '200':
+          description: Service is alive
+  /readiness:
+    get:
+      tags:
+        - Service settings
+      operationId: f8a_admin.api_v1.get_readiness
+      summary: Get job service readiness
+      responses:
+        '200':
+          description: Service is ready
+  '/component-analyses/{ecosystem}/{component}/{version}':
+    get:
+      tags:
+        - Scan Services
+      operationId: f8a_scanner.api_v1.component-analyses
+      summary: Get details about a software component
+      parameters:
+        - in: path
+          name: ecosystem
+          type: string
+          required: true
+          description: component ecosystem
+        - in: path
+          name: component
+          type: string
+          required: true
+          description: component name
+        - in: path
+          name: version
+          type: string
+          required: true
+          description: component version
+      responses:
+        '200':
+          schema:
+            $ref: "#/definitions/ComponentAnalysesResponse"
+          description: Detailed response for the requested application component
+        '401':
+          description: Request unauthorized
+        '404':
+          description: Data not found
+definitions:
+  Version:
+    title: Component Version Details
+    description: Component version details
+    properties:
+      pecosystem:
+        type: array
+        items:
+          type: string
+      pname:
+        type: array
+        items:
+          type: string
+      version:
+        type: array
+        items:
+          type: string
+      declared_licenses:
+        type: array
+        items:
+          type: string
+      licenses:
+        type: array
+        items:
+          type: string
+      last_updated:
+        type: array
+        items:
+          type: number
+          format: float
+      vertex_label:
+        type: array
+        items:
+          type: string
+      cve_ids:
+        type: array
+        items:
+          type: string
+      description:
+        type: array
+        items:
+          type: string
+      cm_loc:
+        type: array
+        items:
+          type: integer
+      cm_num_files:
+        type: array
+        items:
+          type: integer
+      cm_avg_cyclomatic_complexity:
+        type: array
+        items:
+          type: integer
+      shipped_as_downstream:
+        type: array
+        items:
+          type: boolean
+      dependents_count:
+        type: array
+        items:
+          type: integer
+  Package:
+    title: Component Package Specific Details
```

Comment:
Lots of properties are redundant or not maintained. We need to remove the same. Maybe as part of cleanup exercise and proper json response for comp-analyses.